lib/config/config.go

```diff
@@ -56,7 +57,12 @@ var (
 	// DefaultDiscoveryServers should be substituted when the configuration
 	// contains <globalAnnounceServer>default</globalAnnounceServer>.
 	DefaultDiscoveryServers = append(DefaultDiscoveryServersV4, DefaultDiscoveryServersV6...)
-
+	// DefaultStunServers should be substituted when the configuration
+	// contains <stunServer>default</stunServer>.
+	DefaultStunServers = []string{
+		"stun.ekiga.net:3478",
+		"stun1.voiceeclipse.net:3478",
+	}
```

Comment:
We probably need to do something about these, as I am sure we'll  just get banned somehow by this.